hack/virtc/main.go

```diff
@@ -227,7 +228,14 @@ func buildPodConfig(context *cli.Context) (vc.PodConfig, error) {
 		Memory: vmMemory,
 	}
 
+	id := context.String("pod-id")
```

Comment:
Added. This raises the question though of what is considered a valid podID and containerID. Since these end up being encoded in pathnames, we should take care here.

Currently containerID (via `--id=`) is free-form, whereas `podID` default to a `uuid` (although this branch would potentially allow that to change).

@sameo - Could you clarify what the approach should be here by answering a couple of questions:

- should `podID` be restricted to being a `uuid`? I'm guessing yes. If so, maybe we shouldn't allow a user to specify a uuid value manually?
- should `containerID` be restricted to being a `uuid`? I'm guessing no, but how about we restrict the name to only be able to contain characters from the set `[a-ZA-Z0-9-_.]`? (Do we want unicode support?)